packages/docs/.gitignore

```diff
@@ -7,3 +7,4 @@ node_modules
 .env
```

Comment:
Got an error on `serve:docs`.

On `develop` docs works fine.

![image](https://github.com/epicmaxco/vuestic-ui/assets/5394573/c71e6409-d049-49e6-82c0-77522936ecd5)

Reply:
Updating deps and clearing cache might help.

Reply:
yes, i had the same error and have fixed by clearing cookies